src/control/server/ioserver/exec_test.go

```diff
@@ -124,9 +125,13 @@ func TestRunnerContextExit(t *testing.T) {
 	cancel()
 
 	exitErr := <-errOut
-	if errors.Cause(exitErr) == NormalExit {
+	if errors.Cause(exitErr.Err) == NormalExit {
 		t.Fatal("expected process to not exit normally")
 	}
+	if exitErr.Idx != 9 {
```

Comment:
nit - should use cfg.Index instead of the literal 9

Reply:
will do